do not forget to initialize *ngroups argument for getgrouplist()

Fixes spurious gotwebd startup failures.

upstream commit 15cc6a0a0dfee48d9c101a5e4e589e3cb55ad343
Index: gotwebd/gotwebd.c
--- gotwebd/gotwebd.c.orig
+++ gotwebd/gotwebd.c
@@ -357,7 +357,7 @@ main(int argc, char **argv)
 	struct event_base	*evb;
 	struct gotwebd		*env;
 	struct passwd		*pw;
-	int			 ch, i, gotwebd_ngroups;
+	int			 ch, i, gotwebd_ngroups = NGROUPS_MAX;
 	int			 no_action = 0;
 	int			 proc_type = GOTWEBD_PROC_PARENT;
 	const char		*conffile = GOTWEBD_CONF;
